docs/documentation/server_admin/topics/clients/con-protocol-mappers.adoc

```diff
@@ -75,7 +75,7 @@ Further, when the resource server acquires the PII removed from the access token
 
 Information that cannot be removed from a lightweight access token::
   Protocol mappers can controls which information is put onto an access token and the lightweight access token use the protocol mappers. Therefore, the following information cannot be removed from the lightweight access. +
-  `exp`, `iat`, `jti`, `iss`, `sub`, `typ`, `azp`, `nonce`, `sid`, `scope`, `cnf`
+  `exp`, `iat`, `jti`, `iss`, `typ`, `azp`, `nonce`, `sid`, `scope`, `cnf`
```

Comment:
Nitpick: It can be good to remove also `nonce` from this list. It was removed from lightweight access token in some of previous PRs.

However this is not a blocker for this PR as we can do in the follow-up documentation issue we have. Just in case that @andymunro found some other things during docs review.

Reply:
Thanks, removed `nonce` claim.